ui/raidboss/data/07-dt/dungeon/yuweyawata.txt

```diff
@@ -0,0 +1,261 @@
+### YUWEYAWATA FIELD STATION
+# ZoneId: 1242
+
+hideall "--Reset--"
+hideall "--sync--"
+
+# .*is no longer sealed
+0.0 "--Reset--" SystemLogMessage { id: "7DE" } window 0,100000 jump 0
+
+#~~~~~~~~~~~~~~~~~~#
+# Lindblum Zaghnal #
+#~~~~~~~~~~~~~~~~~~#
+
+# -ii 9EB8 9EBA A0A2
+# -it "Raw Electrope"
+# -p 9EBB:1010.1
+
+# The Crystal Quarry will be sealed off
+1000.0 "--sync--" SystemLogMessage { id: "7DC", param1: "1408" } window 1000,1
+1005.1 "--sync--" StartsUsing { id: "9EBB", source: "Lindblum Zaghnal" }
+1010.1 "Electrical Overload" Ability { id: "9EBB", source: "Lindblum Zaghnal" }
+1017.5 "--sync--" Ability { id: "9EAE", source: "Lindblum Zaghnal" }
+1022.7 "Gore" Ability { id: "A132", source: "Lindblum Zaghnal" }
+1032.4 "Line Voltage" Ability { id: "9EB1", source: "Lindblum Zaghnal" }
+1035.4 "Line Voltage" Ability { id: "9EB1", source: "Lindblum Zaghnal" }
+1038.4 "Line Voltage" Ability { id: "9EB1", source: "Lindblum Zaghnal" }
+1043.8 "Caber Toss" Ability { id: "9EB0", source: "Lindblum Zaghnal" }
+1045.8 "Cell Shock" Ability { id: "9EB2", source: "Lindblum Zaghnal" }
+1047.1 "Line Voltage (multiple)" Ability { id: "A0A1", source: "Lindblum Zaghnal" } duration 7.6
+1048.3 "--sync--" Ability { id: "9EB3", source: "Lindblum Zaghnal" }
+1054.9 "--sync--" Ability { id: "9EB3", source: "Lindblum Zaghnal" }
+1057.2 "Lightning Storm" Ability { id: "9EBC", source: "Lindblum Zaghnal" }
+1057.8 "--sync--" Ability { id: "9EBD", source: "Lindblum Zaghnal" }
+1063.6 "--sync--" Ability { id: "9EAE", source: "Lindblum Zaghnal" }
+1068.9 "Gore" Ability { id: "9EB6", source: "Lindblum Zaghnal" }
+1071.0 "Sparking Fissure" Ability { id: "A133", source: "Lindblum Zaghnal" }
+1072.3 "--adds targetable--"
+1079.0 "Sparking Fissure" Ability { id: "9EB7", source: "Lindblum Zaghnal" }
+1081.0 "--adds targetable--"
+1088.1 "Sparking Fissure" Ability { id: "9EB7", source: "Lindblum Zaghnal" }
+1089.1 "Lightning Bolt" #Ability { id: "9EBE", source: "Lindblum Zaghnal" }
+1089.5 "--adds targetable--"
+1090.2 "Lightning Bolt" #Ability { id: "9EBE", source: "Lindblum Zaghnal" }
+1091.2 "Lightning Bolt" #Ability { id: "9EBE", source: "Lindblum Zaghnal" }
+1099.8 "Lightning Bolt" #Ability { id: "9EBE", source: "Lindblum Zaghnal" }
+1100.8 "Lightning Bolt" #Ability { id: "9EBE", source: "Lindblum Zaghnal" }
+1101.8 "Lightning Bolt" #Ability { id: "9EBE", source: "Lindblum Zaghnal" }
```

Comment:
This felt a little cluttered - maybe stick a short duration on the 1st and 3rd entries, and drop 2/4/5?